packages/react/src/components/LightboxMediaViewer/LightboxMediaViewer.js

```diff
@@ -142,7 +142,7 @@ const LightboxMediaViewer = ({ media, onClose, ...modalProps }) => {
    * Stop video on modal close
    */
   function closeModal() {
-    if (onClose?.() !== false) {
+    if (onClose?.() !== false && root.kWidget) {
       root.kWidget.addReadyCallback(videoId => {
```

Comment:
would this also work as expected when Kaltura isn't instantiated?

```suggestion
    if (onClose?.() !== false) {
      root.kWidget?.addReadyCallback(videoId => {
```

Reply:
@emyarod It would, yeah. `onClose` works regardless since it's inherited from `ExpressiveModal`. That condition is just to stop the video when the modal closes so it's not needed for the default version.

Reply:
got it, makes sense!